qiskit/tools/monitor/overview.py

```diff
@@ -96,55 +94,54 @@ def backend_monitor(backend):
         return
 
     print()
-    qubit_header = 'Qubits [Name / Freq / T1 / T2 / U1 err / U2 err / U3 err / Readout err]'
-    print(qubit_header)
-    print('-'*len(qubit_header))
-
+    props = backend.properties()
+    qubit_header = None
     sep = ' / '
-    for qub in range(len(props['qubits'])):
-        name = 'Q%s' % qub
-        qubit_data = props['qubits'][qub]
-        gate_data = [g for g in props['gates'] if g['qubits'] == [qub]]
-        t1_info = qubit_data[0]
-        t2_info = qubit_data[1]
-        freq_info = qubit_data[2]
-        readout_info = qubit_data[3]
-
-        freq = str(round(freq_info['value'], 5))+' '+freq_info['unit']
-        T1 = str(round(t1_info['value'],
-                       5))+' ' + t1_info['unit']
-        T2 = str(round(t2_info['value'],
-                       5))+' ' + t2_info['unit']
-        for gd in gate_data:
-            if gd['gate'] == 'u1':
-                U1 = str(round(gd['parameters'][0]['value'], 5))
-                break
 
-        for gd in gate_data:
-            if gd['gate'] == 'u2':
-                U2 = str(round(gd['parameters'][0]['value'], 5))
-                break
-        for gd in gate_data:
-            if gd['gate'] == 'u3':
-                U3 = str(round(gd['parameters'][0]['value'], 5))
-                break
+    for index, qubit_data in enumerate(props.qubits):
```

Comment:
Here these methods can be used https://qiskit.org/documentation/stubs/qiskit.providers.models.BackendProperties.html#qiskit.providers.models.BackendProperties.gate_property, https://qiskit.org/documentation/stubs/qiskit.providers.models.BackendProperties.html#qiskit.providers.models.BackendProperties.qubit_property to cleanup property finding wrapped in trys for missing properties.

Reply:
`qubit_property()` doesn't give me the unit, and I'd rather extract the unit from the properties than hard coding/making assumptions.

Reply:
`gate_length` and `gate_error` are available for these cases.

Reply:
fixed in 6d3334f